### ISSUE TITLE
[Accessibility] Three finger scrolling does not announce new content 

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#64924]
b/166078444

On iOS, when voiceover is on, swiping with three fingers will scroll a list but won't announce the new content.

The expected behavior is to announce the new content.
http://rcll/gGOBtiISOHpYegP2VDqs8O/c5mHNexqXT3BpPV9xJAwBV [internal only]

Comparison with native list:
http://rcll/gGOBtiISOHpYegP2VDqs8O/d2RkZZDhUx1QdMFfdpdXEI [internal only]

**Comment by goderbauer** [comment]
When clicking on the links I get "You're not authorized to access this page."

**Comment by mehmetf** [comment]
Odd. I used to be able to access these. Asked for access.

**Comment by chunhtai** [comment]
Unassign myself since i don't have time to work on this.  Maybe able to solve by providing UIScrollViewAccessibilityDelegate  to the FlutterScrollableSemanticsObject 

https://developer.apple.com/documentation/uikit/uiscrollviewaccessibilitydelegate?language=objc

**Comment by 838** [comment]
Any plan to look at this issue in future? 🙂
